Update CHANGELOG.md (#386)
eclipse-glsp/glsp: CHANGELOG.md

Introduced reusable view for rounded corner nodes and and improved edge view that supports custom padding for easer mouse handling. Adpated the workflow example to make use of these new views [#180](https://github.com/eclipse-glsp/glsp-client/pull/113)
- [example] Cleaned up and reworked the workflow example. Additional css classes are now applied directly to the `SModelElement` instead of using custom views. Removed now obsolete classes `TaskNodeView` and `WeightedEdgeView` [#220](https://github.com/eclipse-glsp/glsp-client/pull/116)
- [diagram] Fixed a bug in the connection tool regarding the feedback edge snapping computation for nested elements. [#224](https://github.com/eclipse-glsp/glsp-client/pull/123)

> - [diagram] Fixed a bug in the copy& paste behavior. [#249](https://github.com/eclipse-glsp/glsp-client/pull/124

Reviewer: ```suggestion
- [diagram] Fixed a bug in the copy& paste behavior. [#249](https://github.com/eclipse-glsp/glsp-client/pull/124)
```

- [theia-frontend] Cleanup diagram widget initialization by removing no longer needed options [#123](https://github.com/eclipse-glsp/glsp-theia-integration/pull/60)
- [theia-frontend] Fixed a bug that prevented activation of the diagram widget on model source changes [#168](https://github.com/eclipse-glsp/glsp-theia-integration/pull/61)
- [theia-frontend] Fixed a bug that kept the hover feedback visible after the diagram widget becomes inactive [#184](https://github.com/eclipse-glsp/glsp-theia-integration/pull/64)
- [theia-frontend] Made rebind of `CommandPalette` to `TheiaCommandPalette` optional to ensure compatibility with DI configurations where no `CommandPalette` is bound [#188](https://github.com/eclipse-glsp/glsp-theia-integration/pull/65)
- [theia-frontend] Adapted `SetDirtyStateAction` to provide an optional `reason` property indicating the cause for the dirty state change  [#197](https://github.com/eclipse-glsp/glsp-theia-integration/pull/67)
- [theia-fronted] Introduced `GLSPSelectionDataService` which can be used to forward additional information on top of the selection to the Theia selection service. [#228](https://github.com/eclipse-glsp/glsp/issues/228)

> - [theia-frontend] Fixed a bug that displayed the diagram widget as inative when intially openend. [#243](https://github.com/eclipse-glsp/glsp-theia-integration/pull/75)

Reviewer: ```suggestion
- [theia-frontend] Fixed a bug that displayed the diagram widget as inactive when initially openend. [#243](https://github.com/eclipse-glsp/glsp-theia-integration/pull/75)
```

- [glsp-server] Refactored methods in`DiagramConfiguration` to correctly reflect `ShapeTypeHint` instead of `NodeTypeHint`.
- [glsp-client] Fixed the definition of `ChangeContainerOperation`. The type of the `location` property has been changed from `string` to `Point`. [#253 ](eclipse-glsp/glsp-server#115)
- [theia-integration] Refactored the theia-integration code base to remove boilerplate configuration code. This effects the many components of the base API. More details can be found in the corresponding PR. [#258](https://github.com/eclipse-glsp/glsp-theia-integration/pull/84)
- [glsp-server] Update dependencies of Googgle Guava (>= 30.1) and Google Guice (>= 5.0.0). [#260](https://github.com/eclipse-glsp/glsp-server/pull/119)
- [glsp-client] Remove the `name` property from `GLSPClient`. [#258](https://github.com/eclipse-glsp/glsp-client/pull/130/files)
- [glsp-server] Refactor the base communication protocol to support initializing and disposing a client session. Remove now obsolete `InitializeClientSesionAction` and `DisposeClientSessionAction`. [#315](https://github.com/eclipse-glsp/glsp-server/pull/123)

> - [glsp-client] Refactor the base communication protocol to support initializing and disposing a client session. Remove now obsolete `InitializeClientSesionAction` and `DisposeClientSessionAction`. [#315](https://github.com/eclipse-glsp/glsp-client/pull/132)

Reviewer: ```suggestion
```

This is a duplicate with the line above.
